src/Tickets/Blocks/Tickets/app/editor/action-dashboard/template.js

```diff
@@ -67,42 +68,41 @@ class TicketsDashboardAction extends PureComponent {
 			}
 		}
 
-		if (showWarning) {
-			const icon = this.state.isWarningOpen ? 'no' : 'info-outline';
-			const text = this.state.isWarningOpen
-				? __('Hide Warning', 'event-tickets')
-				: __('Warning', 'event-tickets');
-			actions.push(
-				<WarningButton icon={icon} onClick={this.onWarningClick}>
-					{text}
-				</WarningButton>
-			);
-		}
-
 		return actions;
 	};
 
 	render() {
-		const { onConfirmClick, showConfirm } = this.props;
+		const { isBlockSelected, onConfirmClick, showConfirm, showWarning } = this.props;
+
+		const actionDashboardClassName = classNames(
+			'tribe-common',
+			'tribe-editor__tickets__action-dashboard', {
+			'tribe-editor__tickets__action-dashboard__no-border-bottom': showWarning && isBlockSelected,
+		} );
 
 		return (
 			<Fragment>
 				<ActionDashboard
-					className="tribe-common tribe-editor__tickets__action-dashboard"
+					className={actionDashboardClassName}
 					actions={this.getActions()}
 					confirmLabel={confirmLabel}
 					onConfirmClick={onConfirmClick}
 					showCancel={false}
 					showConfirm={showConfirm}
 				/>
-				{this.state.isWarningOpen && (
-					<div className="tribe-editor__tickets__warning">
-						{__(
-							'This is a recurring event. If you add tickets they will only show up on the next upcoming event in the recurrence pattern. The same ticket form will appear across all events in the series. Please configure your events accordingly.', // eslint-disable-line max-len
-							'event-tickets'
-						)}
-					</div>
-				)}
+				{
+					showWarning && isBlockSelected
```

Comment:
Is this change compatible with the existing UI (when Flexible Tickets is not active) for normal posts, Single and Recurring Events?

Reply:
https://www.loom.com/share/f6f5bb3e552447a282b5901e571f132f here is a screencast showing that is compatible